osbs/http.py

```diff
@@ -43,7 +43,7 @@ def decoded_json(iterable):
         if isinstance(line, str):
             yield line
         else:
-            yield line.decode(requests.utils.guess_json_utf(line))
+            yield line.encode(requests.utils.guess_json_utf(line))
```

Comment:
I think the question is to leave method name as `decoded_json` or rename it to `encoded_json`.

I don't have a preference, I'd leave it as is, meaning 'decode from whatever there is - a str in case of py3 or encoded to unicode in py2'